ieeeuoft/makeuoft-site-2021
# 39 user email csv fix

## Overview

- Resolves #39
- Fixed User's Email in the CSV output to be `user.email` instead of just `user`
- Checked using the same steps in QA section that the change was correct.

## Unit Tests Created

- Still none


## Steps to QA

- Go to `Admin Site` -> `Applications`, select which apps you want to download and then download them using the dropdown. Inspect and make sure the email is correct.
